src/components/schema.js

```diff
@@ -17,34 +18,76 @@ const messages = defineMessages({
     id: 'Alignment',
     defaultMessage: 'Alignment',
   },
+  layoutContainer: {
+    id: '100%',
+    defaultMessage: '100%',
+  },
+  left: {
+    id: 'Left',
+    defaultMessage: 'Left',
+  },
+  right: {
+    id: 'Right',
+    defaultMessage: 'Right',
+  },
+  center: {
+    id: 'Center',
+    defaultMessage: 'Center',
+  },
+  innerAlign: {
+    id: ' Inner Alignment',
+    defaultMessage: 'Inner Alignment',
+  },
 });
 
-export const ButtonSchema = (props) => ({
-  title: props.intl.formatMessage(messages.ButtonBlock),
-  block: 'buttonBlock',
-  fieldsets: [
-    {
-      id: 'default',
-      title: 'Default',
-      fields: ['title', 'href', 'align'],
-    },
-  ],
+export const ButtonSchema = (props) => {
+  const { intl } = props;
 
-  properties: {
-    title: {
-      title: props.intl.formatMessage(messages.ButtonTitle),
-    },
-    href: {
-      title: props.intl.formatMessage(messages.ButtonLink),
-      widget: 'object_browser',
-      mode: 'link',
-      selectedItemAttrs: ['Title', 'Description', 'hasPreviewImage'],
-      allowExternals: true,
-    },
-    align: {
-      title: props.intl.formatMessage(messages.Align),
-      widget: 'align',
+  return {
+    title: intl.formatMessage(messages.ButtonBlock),
+    block: 'buttonBlock',
+    fieldsets: [
+      {
+        id: 'default',
+        title: 'Default',
+        fields: ['title', 'href', 'align'],
+      },
+    ],
+
+    properties: {
+      title: {
+        title: props.intl.formatMessage(messages.ButtonTitle),
+      },
+      href: {
+        title: props.intl.formatMessage(messages.ButtonLink),
+        widget: 'object_browser',
+        mode: 'link',
+        selectedItemAttrs: ['Title', 'Description', 'hasPreviewImage'],
+        allowExternals: true,
+      },
+      align: {
```

Comment:
This is the inner align. I've created a special widget for it:

https://github.com/kitconcept/dlr-internet/pull/26/files#diff-074cf83b2f3f969c2555161ff4169fbfae44bd4a303a36efc9864adc5bb4a368R1

Copy it over here, and use it, so the add-on can function without it. We will move it to the core later.

Change the name of the field to `inner_align` and the title in Intl too.